### PR TITLE
Comments presenter

### LINKED ISSUE #359: Comments Presenter
We need `CommentsPresenter`, similar to `AttachmentsPresenter` and `ApplicationsPresenter`, which will show number of comments in the table (and other views) and show last 3 comments in popup on hover

### PR DESCRIPTION
**Pull request by BykhovDenis**
close #359 Signed-off-by: Denis Bykhov [bykhov.denis@gmail.com](mailto:bykhov.denis@gmail.com)

### CODE REVIEW

File: plugins/chunter-resources/src/components/CommentPresenter.svelte [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+  }
+
+  function isToday (time: number): boolean {
+    const current = new Date()
+    const target = new Date(time)
+    return current.getDate() === target.getDate() && current.getMonth() === target.getMonth() && current.getFullYear() === target.getFullYear()
+  }
+</script>
+
+<div class="container">
```
Comment (haiodo) [inline plugins/chunter-resources/src/components/CommentPresenter.svelte:53]: Why not reuse existing Comment component?

**Review by haiodo — COMMENTED** [review]
I've don't see an server code to update comments counter, or it was done before?

File: models/recruit/src/index.ts [hunk 2]
```diff
@@ -46,6 +46,9 @@ export class TCandidate extends TPerson implements Candidate {
   @Prop(TypeString(), 'Attachments' as IntlString)
   attachments?: number
 
+  @Prop(TypeString(), 'Comments' as IntlString)
+  comments?: number
```
Comment (haiodo) [inline models/recruit/src/index.ts:50]: I've could not find there is it updated?

File: plugins/chunter-resources/src/components/CommentPresenter.svelte (hunk 1 above)
Comment (BykhovDenis) [reply plugins/chunter-resources/src/components/CommentPresenter.svelte:53]: A strange component, I did not find it to be used, I think it needs to be removed

File: models/recruit/src/index.ts (hunk 2 above)
Comment (BykhovDenis) [reply models/recruit/src/index.ts:50]: server/core/storage:109 here

**Review by aplatoff — APPROVED** [review]

**Pull request merged**